packages/core/custom-elements-lsp/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { init } from './plugin';
+
+module.exports = init;
```

Comment:
Mixing ES6 and CommonJS in the same module - should use one or the other

Reply:
Will get to this another time but going to merge to try out the publishing now. Thanks.